erpnext_autoinstall/install.py

```diff
@@ -0,0 +1,75 @@
+import os
+
+import frappe
+
+
+def after_install():
+    disable_registration()
+    print("REGISTRATION HAS BEEN DISABLED")
+    configure_email()
+    print("EMAIL HAS BEEN CONFIGURED")
+    configure_ldap()
+    print("LDAP HAS BEEN CONFIGURED!")
```

Comment:
@AminovE99 Good idea to print what configuration has been done, but it should be inside each function, so if configuration has been actually done.